### PR TITLE
Add convert .docx to .pdf files feature and exception handling

### PR DESCRIPTION
**Pull request by luisdavidparra**

### CODE REVIEW

File: CONVERTER-SERVICE/.env [hunk 1]
```diff
@@ -1,3 +1,8 @@
 PORT=9090
 URL='/api/v1/'
-FORMATSUPPORTED = ['mov', 'm4a', '3gp', '3g2', 'mj2', 'mp4', 'jpeg', 'png'];
\ No newline at end of file
+URLBASE = 'http://localhost:'
+FORMATS_SUPPORTED_VIDEO = ['mov', 'm4a', '3gp', '3g2', 'mj2', 'mp4'];
+FORMATS_SUPPORTED_IMAGE = ['jpeg', 'png', 'jpg'];
+FORMATS_SUPPORTED_DOC = ['docx'];
+NOT_FOUND_ERROR_CODE = '404';
+BAD_REQUEST_ERROR_CODE = '400';
```
Comment (stospina) [inline CONVERTER-SERVICE/.env]: Newline at the end

File: CONVERTER-SERVICE/.gitignore [hunk 2]
```diff
@@ -1,5 +1,3 @@
 node_modules
-*.mp4
-*.jpg
-*.png
-*.zip
\ No newline at end of file
+files/downloadFiles/*
+files/uploads/*
```
Comment (stospina) [inline CONVERTER-SERVICE/.gitignore]: Newline at the end

**Review by stospina — COMMENTED** [review]
Looks good

File: CONVERTER-SERVICE/src/Exceptions/converter.exception.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+// Exception for converter methods
+class ConverterException extends ServerException {
+
+  constructor(message, code){
+    super(message, '404' , code);
+  }
+}
+
+module.exports = ConverterException;
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/Exceptions/converter.exception.js:23]: leave an empty line at the end

File: CONVERTER-SERVICE/src/Exceptions/invalidFile.exception.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+// Exception for invalid files
+class InvalidFileException extends ServerException {
+
+  constructor(message, code){
+    super(message, '400', code);
+  }
+}
+
+module.exports = InvalidFileException;
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/Exceptions/invalidFile.exception.js:23]: leave an empty line at the end

File: CONVERTER-SERVICE/src/controllers/downloadFile.controller.js [hunk 5]
```diff
@@ -10,15 +10,17 @@ accordance with the terms of the license agreement you entered into
 with Jalasoft.
 */
 
-const fs = require('fs');
-
-module.exports = class DownloadController {
-    
+class DownloaFileController {
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/controllers/downloadFile.controller.js:13]: comment before class

File: CONVERTER-SERVICE/src/controllers/imageConverter.controller.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const fs = require('fs');
+const Upload = require('../helpers/upload.helper');
+const ImageConverter = require('../models/imageConverter.model');
+const Converter = require('./converter.controller');
+const { PORT, URL, URLBASE } = process.env;
+
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/controllers/imageConverter.controller.js:17]: add a comment before tha class

File: CONVERTER-SERVICE/src/controllers/videoConverter.controller.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+*/
+
+const Upload = require('../helpers/upload.helper');
+const Compress = require('../helpers/compress.helper');
+const VideoConverter = require('../models/videoConverter.model');
+const Converter = require('./converter.controller');
+const fs = require('fs');
+const { PORT, URL, URLBASE } = process.env;
+
+class VideoConverterController extends Converter {
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/controllers/videoConverter.controller.js:21]: add a comment before the class

**Review by florpadilla — COMMENTED** [review]
check the comments of the classes and the empty lines at the end

**Review by stospina — COMMENTED** [review]
Everything looks good

**Pull request merged**